src/cached_object_store.rs

```diff
@@ -53,10 +64,35 @@ impl CachedObjectStore {
             scan_interval,
             db_stats.clone(),
         ));
+
+        // If `max_cache_size_bytes` is None, the initial capacity of ghost queue is 32 entries and capacity will
+        // adjust from 32 to 64 so that we can provide some resistance even if there is no eviction on disk cache
+        //
+        // Otherwise, the initial capacity of ghost queue is `max_cache_size_bytes / 10`, and we will enable
+        // size aware eviction to adaptively provide relevant resistance
+        // TODO(asukamilet): Consider the case where `max_cache_bytes_size` is `Some(0)` which means we disable the disk cache
+        let ghost = max_cache_size_bytes.map_or(
```

Comment:
`32-64` are just magic numbers. This may be a potential optimization in the future, such as using miss ratio curve to guide capacity.
References: 
1. [Reducing Cross-Cloud/Region Costs with the Auto-Configuring MACARON Cache](https://www.pdl.cmu.edu/ftp/CloudComputing/sosp24-final499.pdf)
2. [Kosmo: Efficient Online Miss Ratio Curve Generation for Eviction Policy Evaluation](https://www.usenix.org/conference/fast24/presentation/shakiba)